health/drivers/tcp/driver.py

```diff
@@ -154,8 +155,15 @@ def main(es, latest_aggregated_ts=None):
 
     for interval in intervals:
         body = get_request(interval)
-        resp = requests.post("%s/_search?search_type=count" % es,
-                             data=json.dumps(body)).json()
+        resp = requests.post("%s/_search" % es,
```

Comment:
so where do we specify that it's search_type=count? how elastic understands to not return all values?